### ISSUE TITLE
USB Boot issue

### ISSUE THREAD

**Issue by dragonsoulpl** [tlefko/Matebook-X-Pro-2018-Ventura#30]
Hi tlefko,
I'm currently in the process of making my 2018 MxP a hackintosh. I've run into an issue where whenever I boot into my usb drive, it will boot into OpenCore, but the only option is "UEFI Shell". Any idea as to why this is happening? I believe the solution is more simple than I think but my knowledge in this topic is limited.

EDIT: Now that I'm looking back at it, I do not even have an option for windows, but booting straight into windows still works

**Comment by tlefko** [comment]
@dragonsoulpl sounds like it is not detecting the drives, did you format the drive as macOS journaled? maybe check the security settings within opencore, which verison of the EFI are you using?

**Comment by dragonsoulpl** [comment]
I downloaded the zip from this repo for my files. I used gib to create the boot usb. Replaced the OC folders inside and added the config plist. Is there a way for me to IM you if you are available by any chance?

**Comment by tlefko** [comment]
> @tlefko
> I downloaded the zip from this repo for my files. I used gib to create the boot usb. Replaced the OC folders inside and added the config plist. Is there a way for me to IM you if you are available by any chance?

Yes, you can add my instagram, Tyler.lefko and we can discuss.

**Comment by Paxolo** [comment]
The new EFI isn't working for me either.
I even tried with a clean installation (sadly with no luck) so I ended up reinstalling macOS using the previous release.
I find it very unlikely to be a problem with the USB thumb as I used the same one with both of the EFI folders.

**Comment by tlefko** [comment]
@Paxolo would you mind providing some more details?

**Comment by Paxolo** [comment]
> @Paxolo would you mind providing some more details?

I basically had the exact same experience as @dragonsoulpl 

I already had macOS installed, so I thought I could just copy-paste the new EFI folder in my EFI partition and all I had to do was keeping my serial number, etc.
And so I did, I downloaded the latest release and used MountEFI to put it into the EFI partition of my drive, then restarted.
It reached the OpenCore selector successfully, but my only boot option was "UEFI shell". 
I tried pressing the spacebar in case my macOS drive was being detected as "auxiliary", but nothing happened.
At this point I just switched back to the EFI I had already been using for some time
